tests/utils.py

```diff
@@ -198,3 +205,19 @@ def see_dashboard_role_perms(self):
     @property
     def see_dashboard_for_all_orgs_perms(self):
         return []
+
+
+def _get_url_info(url):
+    parsed_url = urlparse(url)
+    url_path = parsed_url.path
+    url_query = set(parsed_url.query.split("&"))
+    return url_path, url_query
+
+
+def find_url(url, urls):
+    main_url_path, main_url_query = _get_url_info(url)
+    for url_check in urls:
+        check_url_path, check_url_query = _get_url_info(url_check)
+        if check_url_path == main_url_path and check_url_query == main_url_query:
+            return url_check
+    return None
```

Comment:
Where are these used?

Reply:
I have used find_url https://github.com/open-craft/xblock-group-project-v2/pull/142/files#diff-98567b7d0375fc360340639185c12f7bR40 

and `_get_url_info` is used inside it